js/src/scrollspy.js

```diff
@@ -58,16 +56,17 @@ const DefaultType = {
 class ScrollSpy extends BaseComponent {
   constructor(element, config) {
     super(element, config)
-    this._scrollElement = this._element.tagName === 'BODY' ? window : this._element
-    this._offsets = []
-    this._targets = []
-    this._activeTarget = null
-    this._scrollHeight = 0
 
-    EventHandler.on(this._scrollElement, EVENT_SCROLL, () => this._process())
-
-    this.refresh()
-    this._process()
+    // this._element is the observablesContainer and config.target the menu links wrapper
```

Comment:
This comment might be outdated

Reply:
It is on purpose to explain the usage of the variables. (When I started to refactor the code, it wasn't easy to understand the usage of theses variables. I thought  it will help some future contributor

Reply:
I don't mind explaining, it's that `this._element` is not set here anymore :P

Reply:
I know :/, if you have any better suggestion, please apply it